dummy_lib/core.py

```diff
@@ -0,0 +1,5 @@
+__all__ = ["greet_contributor"]
+
+
+def greet_contributor(name):
```

Comment:
Would you mind adding type annotations? (cf. https://www.stackbuilders.com/blog/using-types-in-python-with-mypy/)

What about:
```suggestion
def greet_contributor(name: str) -> str:
```

This feedback is about: **[understandability] [type annotations]**. If this is wrong, click [here](url).